charts/secrets-store-csi-driver-provider-aws/templates/daemonset.yaml

```diff
@@ -31,6 +31,8 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - --provider-volume={{ .Values.providerVolume }}
+            - --qps={{ .Values.k8sThrottlingParams.qps | default 5 }}
```

Comment:
Can we just not set these args if the value doesn't exist? This saves us from having to update the default value in multiple spots if it ever needs to be adjusted.